src/utils.tsx

```diff
@@ -88,6 +90,42 @@ export const formatDateTimeStrings = (
   return formattedDate;
 };
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const OverflowTip = ({ children }: any) => {
+  const [isOverflowed, setIsOverflow] = React.useState(false);
+  const overflowElementRef = useRef<HTMLInputElement | null>(null);
+  React.useEffect(() => {
```

Comment:
If you use the column options to reset the column size it doesn't update this e.g.
![tooltipBug](https://github.com/ral-facilities/inventory-management-system/assets/90245114/1b198fb1-53f6-4f3e-829b-f06e0442d619)